docs/Basics/01-working-with-projects.md

```diff
@@ -8,12 +8,14 @@ Projects in Studio provide a place for you to manage collections of APIs, articl
 
 ![Studio New Project Screen](../../assets/images/studio-project-screen.png)
 
+A project can be a Git repository, or if you're using Studio Desktop it could just be any local folder on your computer.
+
 ## Project Structure
 
-Studio projects default to using a specific structure to not interfere with existing files. This default structure is:
+Studio projects default to using a specific structure to not interfere with existing files.
```

Comment:
How about rephrasing this as a common convention or a best practice? It's not just a default, it's how we are recommending the files are organized. Though we do allow you to customize it to your own needs and conventions by creating a Stoplight configuration file 🤓

Reply:
Suggesting it as a common convention outside of Stoplight? It's not 😅 

Suggesting it's one inside Stoplight? Well yeah we used to enforce it. 😆